common/main.c

```diff
@@ -14,4 +16,5 @@ main(void)
 {
 	console_init(DEV_UART0);
 	dm_init();
+	process_workers();
```

Comment:
You need call the function in an infinite loop here.

Reply:
Done